patient-info-backend/src/chat/image/image.controller.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Post,
+  HttpStatus,
+  UseInterceptors,
+  UploadedFile,
+  Controller,
+  Res,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { ApiResponse } from '@nestjs/swagger';
+import { ImageService } from './image.service';
+
+@Controller('image')
+export class ImageController {
+  constructor(private readonly uploadImageService: ImageService) {}
+
+  @Post('')
+  @UseInterceptors(FileInterceptor('image', { limits: { files: 1 } }))
+  @ApiResponse({ status: HttpStatus.CREATED })
```

Comment:
what status is sent if this line removed?

Reply:
This line is of swagger. Its like a documentation of what response would look like